rtmp-stream.go

```diff
@@ -98,6 +114,67 @@ func (f *RtmpStreamFactory) startStream(conn rtmp.ClientConn, streamName string)
 	}
 }
 
+func (f *RtmpStreamFactory) ParseURLArgument(urlArg string) (string, []*url.URL, error) {
+	var host string
+	var urls []*url.URL
+	var r = regexp.MustCompile(`{{(?P<min>[1-9][0-9]*) (?P<max>[1-9][0-9]*)}}`) // {{123 456}}
+
+	var regexString = "'{{(?P<min>[1-9][0-9]*) (?P<max>[1-9][0-9]*)}}'"
+	var regexInfo = fmt.Sprintf("Use regex that matches patter %v", regexString)
+
+	if r.MatchString(urlArg) { // URL is a template.
+		log.Infoln("Processing template URL %v with regex matching. (Used regex: '%v')", urlArg, regexString)
```

Comment:
Use log.Infof instead of Infoln when using formatting sequences (also below, check all log calls).